airflow/www/views.py

```diff
@@ -2244,6 +2244,7 @@ def action_varexport(self, items):
 
         response = make_response(json.dumps(var_dict, sort_keys=True, indent=4))
         response.headers["Content-Disposition"] = "attachment; filename=variables.json"
+        response.headers["Content-Type"] = "application/json"
```

Comment:
```suggestion
        response.headers["Content-Type"] = "application/json; charset=utf-8"
```
I am not sure about it, but the following information is written on the W3C website:
> It is very important to always label Web documents explicitly.

Reply:
Updated

Reply:
Ohh. I missed link: https://www.w3.org/International/articles/http-charset/index